rholang/src/main/scala/coop/rchain/rholang/interpreter/normalize.scala

```diff
@@ -58,10 +58,18 @@ object NameNormalizeMatcher {
         }
 
       case n: NameQuote => {
+        def collapseQuoteEval(p: Par): Channel = {
+          p.singleEval() match {
+            case Some(Eval(chanNew)) => chanNew
```

Comment:
I worry about `@{new x in *y}` vs `@*y`.  Even if `y` gets replaced by something, it can't possibly refer to the new `x`.  But I guess if we're modeling scope manipulations as explicit rewrites (Greg's favorite of the options so far) then they really are different terms.